api/raml/device-sdk.raml

```diff
@@ -0,0 +1,272 @@
+#%RAML 0.8
+---
+title: device-sdk
+version: "1.1.x"
+baseUri: "http://device-x:49999/api"
+protocols: [ HTTP ]
+
+documentation:
+    - title: Device Service API
+      content: This is the REST API implemented by device-sdk-go and device-sdk-c. It is therefore available on device services built with either SDK.
+
+schemas: 
+    -
+        responseobjects: '{"type":"array","$schema":"http://json-schema.org/draft-06/schema#","title":"responseobjects","items":{"type":"object","required":false,"$ref":"#/schemas/responseobject"}}'
+    -
+        responseobject: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"responseobject","properties": {"device":{"description":"the name of the device","type":"string"}, "origin":{"description":"timestamp indicating when the readings were taken","type":"integer"}, "readings":{"type":"array","items":{"type":"object","$ref":"#/schemas/reading"}}}}'
+    -
+        reading: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"reading","properties":{"name":{"type":"string"},"value":{"type":"string"}}}'
+    -
+        setting: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"setting","additionalProperties":{"type":"string"}}'
+    -
+        callbackalert: '{"type": "object","$schema": "http://json-schema.org/draft-06/schema#","title": "Notification Schema","properties": {"id": {"description": "the identifier of the object which is called back","type": "string"},"actionType": {"description": "the type of the called back object","enum":["PROFILE","DEVICE","PROVISIONWATCHER","SCHEDULE","SCHEDULEEVENT"],"type": "string"}},"required": ["id"]}'
+    -
+        versionobject: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"version","properties":{"version":{"type":"string"},"sdk_version":{"type":"string"}}}'
+    -
+
+/version:
+    displayName: Version Strings
+    description: Example -- http://localhost:49999/api/version
+    get:
+        description: Report service and SDK versions
+        responses:
+            "200":
+                body:
+                    application/json:
+                        schema: versionobject
+                        example: '{"version":"1.0","sdk_version":"1.1.0"}'
+
+/v1/ping:
+    displayName: Ping Resource
+    description: Example -- http://localhost:49999/api/v1/ping
+    get:
+        description: Test service providing an indication that the service is available.
+        displayName: service up check
+        responses:
+            "200":
+                body:
+                    text/plain:
+                        description: device service version string
+                        example: 'V1.0'
```

Comment:
in device-sdk-go, it only returns 1.0.0 without "V".
does C SDK returns the version number with "V"?